packages/backend/src/App.ts

```diff
@@ -502,6 +502,10 @@ export default class App {
                     email: req.user.email,
                     username: req.user.email,
                 });
+                Sentry.setTags({
+                    'user.uuid': req.user.userUuid,
+                    'organization.uuid': req.user.organizationUuid,
+                });
```

Comment:
can these serUser and setTags be moved to the sentry middleware? 🤔 so we have one file that does it all?